### ISSUE TITLE
Flash session is nil when doing redirect.

### ISSUE THREAD

**Issue by donperi** [lotus/controller#95]
Hello everyone.

I'm having an issue with flash session. I'm setting the flash session and then using a redirect_to.
In the next request I'm expecting to get the value but is returning nil.

```ruby
  def call(params)
      @organization = Organization.new(params[:organization])

      if @repository.persist(@organization)
        flash[:success] = "Organization created"
        redirect_to routes.path(:manager_organizations)
      else 
        flash[:errors] = @errors = @organization.errors
      end
  end
```

In the view:
```haml
 - if flash[:success]
   = flash[:success]
```

The code is working when the flash is used in the same request. (Using the flash[:errors] in this case.)

**Comment by jodosha** [comment]
@donperi Hello and thanks for opening this issue. :smile: Have you activated sessions?

Flash is marked as private API for now, because we know that it's unstable. Would you love to take care of this? Please let me know if that is the case. Thank you :smile:

**Comment by donperi** [comment]
Yes, sessions are activated via 

```
sessions :cookie, secret: 'SECRET'
```
and it's working fine if I use sessions.

Yes. I can try to take care of it. This would be a good opportunity to share code :smile:

I was reading the code and I think the source of the problem is the request_id key for flash session.

**Comment by jodosha** [comment]
@donperi Interesting. Any help is appreciated. Thank you.

**Comment by jodosha** [comment]
Closed by #96
